src/App.js

```diff
@@ -2,44 +2,44 @@ import './styles/reset.css'
 import './styles/index.css'
 
 import initialStoreItems from './store-items'
-
-/*
-Here's what a store item should look like
-{
-  id: '001-beetroot',
-  name: 'beetroot',
-  price: 0.35
-}
-
-What should a cart item look like? 🤔
-*/
-
-console.log(initialStoreItems)
+import { useEffect, useState } from 'react'
+import Store from './Store'
+import Cart from './Cart'
+import Filters from './Filters'
+import Total from './Total'
 
 export default function App() {
-  // Setup state here...
+  const [items, setItems] = useState(initialStoreItems)
+  const [cart, setCart] = useState([])
+
+  
 
   return (
     <>
       <header id="store">
         <h1>Greengrocers</h1>
+        <Filters
+          setItems={setItems}
+          initialStoreItems={initialStoreItems}
+          items={items}
+        />
         <ul className="item-list store--item-list">
-          {/* Wrtite some code here... */}
```

Comment:
Perhaps try to add some conditional rendering here, if you get a chance to try extension two!